app/pages/project/about/index.spec.js

```diff
@@ -9,6 +9,8 @@ function Page() {
   );
 }
 
+// TODO: THIS BREAKS EVERYTHING, WHY?
```

Comment:
Is this comment still relevant?

Reply:
ah no, it was breaking when I was trying to configure enzyme in setup.js, fixed by making sure globals were set before enzyme configuring.